.golangci.yml

```diff
@@ -1,12 +1,11 @@
 linters-settings:
   gofumpt:
-    lang-version: '1.20'
+    simplify: true
```

Comment:
Analogous to `gofmt -s`, added this extra improvement but can get rid of it unless wanted.